test/TestProjects/CollapseRequestCondition-LowLevel/Generated/MatchConditionCollapseClient.cs

```diff
@@ -267,23 +267,6 @@ internal HttpMessage CreateCollapseGetRequest(MatchConditions matchConditions, R
             return message;
         }
 
-        internal HttpMessage CreateMulticollapseGetRequest(RequestConditions requestConditions, RequestContext context)
-        {
-            var message = _pipeline.CreateMessage(context);
-            var request = message.Request;
-            request.Method = RequestMethod.Get;
-            var uri = new RawRequestUriBuilder();
-            uri.Reset(_endpoint);
-            uri.AppendPath("/MatchConditionCollapse/multi", false);
-            request.Uri = uri;
-            if (requestConditions != null)
-            {
-                request.Headers.Add(requestConditions, "R");
-            }
-            message.ResponseClassifier = ResponseClassifier200.Instance;
-            return message;
-        }
-
```

Comment:
Why is this method got removed? I think you will get compile time error because we are calling this method from  [MulticollapseGetAsync()](https://github.com/Azure/autorest.csharp/blob/cf0fce46867d697a0e6fa91ea9525b0c370bfbad/test/TestProjects/CollapseRequestCondition-LowLevel/Generated/MatchConditionCollapseClient.cs#L183) and [MulticollapseGet()](https://github.com/Azure/autorest.csharp/blob/cf0fce46867d697a0e6fa91ea9525b0c370bfbad/test/TestProjects/CollapseRequestCondition-LowLevel/Generated/MatchConditionCollapseClient.cs#L203).

Reply:
resolved

Reply:
resolved